src/components/fields/single-line-string.js

```diff
@@ -49,6 +49,7 @@ export default createReactClass({
         autoComplete={field.autoComplete}
         autoFocus={field.autoFocus}
         className={cx(this.props.classes)}
+        disabled={field.isDisabled}
```

Comment:
I prefer the `is` prefix but when in Rome, I’d say to follow the convention of not using the prefix for field properties.

Reply:
To match readOnly, we probably should also wrap in a plug-in method, but we can always wrap later if needed too.

Reply:
I think I did the plug-in method correctly, but double-check me on this one